doc/api/inspector.md

```diff
@@ -121,9 +121,15 @@ session.on('Debugger.paused', ({ params }) => {
 added: v8.0.0
 -->
 
-Connects a session to the inspector back-end. An exception will be thrown
-if there is already a connected session established either through the API or by
-a front-end connected to the Inspector WebSocket port.
+Connects a session to the inspector back-end.
+
+### session.connect()
```

Comment:
```suggestion
### session.connectToMainThread()
```

Reply:
Also, to clarify: This connects to the *parent* thread and not necessarily the *main* thread, right?

Reply:
Right! :)

Reply:
Sorry, replied to a different thread.

Current implementation does not support "worker trees" - e.g. only workers with the main thread as a parent are reported by the NodeWorker domain. Is workers spawning workers intenteded use-case? Then it will need to be fixed and I expect that the fix will make this API work in a consistent way.

Reply:
> Is workers spawning workers intenteded use-case?

Yes, that’s possible, mostly because there’s no real reason to prevent it.

Reply:
Then, I'll try fixing `NodeWorker` domain. This API will be consistent with what that domain reports.

Reply:
This is the proposed fix: https://github.com/nodejs/node/pull/28872